pkg/apis/events/v1alpha1/test_helper.go

```diff
@@ -0,0 +1,49 @@
+package v1alpha1
```

Comment:
Similar comment as above, this is now in our production code. Can we move it to a different package?

Reply:
done